src/datasets/pings.md

```diff
@@ -31,3 +145,52 @@ As of September 28, 2018, members of the `meta` key on main pings include:
 <!-- table generated via `scripts/new_ping_metadata_table.py > src/cookbooks/new_ping_metadata_table.md` -->
 
 {{#include ../cookbooks/new_ping_metadata_table.md}}
+
+## Analysis
+
+The large majority of analyses can be completed using only the
+[main ping][main_ping].
+This ping includes histograms, scalars, and other performance and diagnostic data.
+
+Few analyses actually rely directly on any raw ping data.
+Instead, we provide [derived datasets](./derived.md) which are processed versions of these data,
+made to be:
+
+- Easier and faster to query
+- Organized to make the data easier to analyze
+- Cleaned of erroneous or misleading data
```

Comment:
Not to keep hammering this but I think we should avoid implying that e.g. clients_daily is generally "safe". I buy 1 and 2, though.

Reply:
Maybe we should just take out 3?

```suggestion
```

Reply:
sgtm!